<launch>
  <env name="PYTHONPATH"  value="$(env PYTHONPATH)"/> 
  <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):/home/marcela/naoqi/naoqi-sdk-1.14.5-linux64/lib"/>

<<<<<<< HEAD:launch/teleoperation_matrix.launch
  <arg name="fixed_frame" default="openni_depth_frame" />
=======
  <param name="naoqi_ip" value="192.168.0.41" />
  <param name="naoqi_port" value="9559"/>

  <arg name="fixed_frame" default="openni_depth_frame" />


>>>>>>> 4515644f04a904632f51b88eee68e7e64456ba80:launch/kinect_to_nao_publisher.launch
  
  <node pkg="tg_nao_teleoperation" name="kinect_to_nao_publisher" type="kinect_to_nao_publisher_inv.py" output="screen">
  <rosparam file="$(find tg_nao_teleoperation)/parameters/kinect_publisher_parameters.yaml" command="load" />
  <param name="fixed_frame" value="$(arg fixed_frame)" />
  </node>
  
  <node pkg="tg_nao_teleoperation" name="control_nao_arms" type="nao_subscriber_inv.py" output="screen">
  <rosparam file="$(find tg_nao_teleoperation)/parameters/nao_subscriber_parameters.yaml" command="load" />
  </node>
  
</launch>
